editor/src/components/canvas/canvas-strategies/strategies/group-like-helpers.ts

```diff
@@ -162,3 +167,33 @@ export function treatElementAsContentAffecting(
 ): boolean {
   return getElementContentAffectingType(metadata, allElementProps, path) != null
 }
+
+export const GroupFlagKey = 'data-group'
+
+export function isElementMarkedAsGroup(
+  metadata: ElementInstanceMetadataMap,
+  path: ElementPath,
+): boolean {
+  const instance = MetadataUtils.findElementByElementPath(metadata, path)
+  if (instance == null || isLeft(instance.element)) {
+    return false
+  }
+
+  if (isJSXConditionalExpression(instance.element.value)) {
+    return findUtopiaCommentFlag(instance.element.value.comments, 'group')?.value === true
+  }
+
+  if (isJSXElement(instance.element.value)) {
+    return (
+      instance.element.value.props.find(
+        (prop) =>
+          prop.type === 'JSX_ATTRIBUTES_ENTRY' &&
+          prop.key === GroupFlagKey &&
+          prop.value.type === 'ATTRIBUTE_VALUE' &&
+          prop.value.value === true,
+      ) != null
```

Comment:
I'm sure we have a function for checking the value of a prop

Reply:
oh crap I totally missed this! yes we have very good helpers for this :)

Reply:
I wish we had chatgpt or something to pick this up automatically 😅 any pointers about what this function might be?

Reply:
Ok I think the best function to use here is [`getSimpleAttributeAtPath`](https://github.com/concrete-utopia/utopia/blob/master/editor/src/core/model/element-metadata-utils.ts#L2304-L2323>)

Reply:
thanks! added in https://github.com/concrete-utopia/utopia/pull/3504/commits/e8296ed05698f78e61c9aaece04d7eea74524218